packages/patrol_cli/lib/src/crossplatform/app_options.dart

```diff
@@ -115,14 +115,22 @@ class AndroidAppOptions {
     final target = '-Ptarget=${flutter.target}';
     cmd.add(target);
 
+    // Create modifiable Map
+    final effectiveDartDefines = Map<String, String>.of(flutter.dartDefines);
+
+    // Add flavor to dart defines
+    if (flutter.flavor != null) {
+      effectiveDartDefines['FLUTTER_APP_FLAVOR'] = flutter.flavor!;
+    }
```

Comment:
Nit:
```suggestion
    if (flutter.flavor case final flavor?) {
      effectiveDartDefines['FLUTTER_APP_FLAVOR'] = flavor;
    }
```
To get rid of bang operator

Reply:
Thanks for your suggestion.